.github/workflows/NativeTests.yaml

```diff
@@ -0,0 +1,121 @@
+name: NativeTests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths-ignore:
+    - 'spring-cloud-generator/**'
+    - 'spring-cloud-previews/**'
+  workflow_dispatch:
+
+
+jobs:
+  NativeTests:
+    if: |
+      github.actor != 'dependabot[bot]' && ((
```

Comment:
Is there a reason for this limitation? I would think we want to run native tests when updating dependencies.

Reply:
AFAIK, we exclude dependency upgrades or any other pr from outside for security reasons.